datahub-web-react/src/app/lineage/utils/columnLineageUtils.ts

```diff
@@ -71,10 +71,10 @@ export function convertInputFieldsToSchemaFields(inputFields?: InputFields) {
  */
 export function getPopulatedColumnsByUrn(
     columnsByUrn: Record<string, SchemaField[]>,
-    fetchedEntities: { [x: string]: FetchedEntity },
+    fetchedEntities: Map<string, FetchedEntity>,
```

Comment:
nice - okay change below i think would be safe if this is what is getting passed